tests/test_formatter.py

```diff
@@ -203,3 +203,100 @@ def test_escape(self):
 
         # Escaping is atomic: brackets inside brackets are treated literally
         assert self.formatter.format(datetime(1, 1, 1), "[[[ ]]") == "[[ ]"
+
+
+class TestDateTimeFormatterPremadeFormats:
+    def setup_class(cls):
+        cls.formatter = formatter.DateTimeFormatter()
+
+    # cookie format: 'Thursday, 25-Dec-1975 14:15:16 EST'
+    def test_cookie(self):
+        for full_name in make_full_tz_list():
```

Comment:
There's no need to test against the full tz list here, since those tests already happen in `test_timezone_formatter`.